index.html

```diff
@@ -1,14 +1,201 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="/dist/css/styles.css">
-    <title>Document</title>
-</head>
-<body>
-    <h1>Test</h1>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/dist/css/styles.css" />
+    <title>Social Media app</title>
+  </head>
+  <body>
+    <nav class="navbar navbar-light bg-light">
+      <div class="container-fluid">
+        <a class="navbar-brand" href="index.html">
+          <img
+            src="/images/placeholder.com-logo1.png"
+            alt=""
+            width="150"
+            height="35"
```

Comment:
```suggestion
```

Setting height and width for images is not recommended. One of these should be left as `auto` otherwise the image may be rendered with the wrong aspect ratio